docs/tools/new_tools_temp.md

```diff
@@ -0,0 +1,53 @@
+# New tools - draft
+
+<!--well this is a disaster-->
+
+Starting with version 6.x, Mbed OS is moving from Mbed CLI to a new build tool: it's cmake-based,
+
+You'll still need Mbed CLI for older versions of Mbed OS. You can install both tools side by side.<!--seems like Mbed CLI also requires Python 3, so there shouldn't be any issues-->
+
+
+# Install or upgrade
+
+Mbed Tools is Python based, so you can install it with pip.
+<!--other than being a super vague name, it's a problem that it's a single tool with a plural name. "Mbed Tools is"  just parses as an error!-->
+
+<span class="tips">**Tip:** We recommend using a virtual environment such as [Pipenv](https://github.com/pypa/pipenv/blob/master/README.md) to avoid Python dependency conflicts.</span>
```

Comment:
`pipenv` isn't a virtual environment, it's a virtual environment management tool.

Reply:
Do we have a tool we want to recommend, then?

Reply:
We do want to recommend `pipenv` to manage the development environment, at the moment, but we don't want to refer to it as "a virtual environment". A virtual environment is a self-contained directory tree that contains a Python installation for a particular version of Python, plus a number of additional packages. `pipenv` is a "virtual environment management tool".